src/VerifyWindowsHelloController.js

```diff
@@ -94,22 +113,88 @@ function (Okta, FormController, FormType, webauthn, FooterSignout, WindowsHelloE
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-default">\
+              {{i18n code="verify.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-loading">\
+              {{i18n code="verify.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-success">\
+              {{i18n code="verify.windowsHello.subtitle.signingIn" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'success'}
+          }));
+        }
+
+        result.push(FormType.View({View: Spinner}));
+
         return result;
       },
 
+      postRender: function () {
+        if (this.options.appState.get('factors').length === 1 && !this.model.get('__enrollmentState__')) {
+          this.model.save();
```

Comment:
I think that to know that the factor is the "default" one, we need to verify that it is the first one on the list of factors we get back from the API... not sure we should do that here and behave differently if it is the first one as opposed to be in any other position on the list. @rchild-okta do you have any opinion on this one?